src/components/v5/common/ActionSidebar/partials/GroupedActionList/GroupedActionList.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { type FC } from 'react';
+
+import {
+  GroupedActionItem,
+  type GroupedActionItemProps,
+} from './GroupedActionItem.tsx';
+
+interface GroupedActionListProps {
+  items: Omit<GroupedActionItemProps, 'color'>[];
+  color: GroupedActionItemProps['color'];
+}
+
+export const GroupedActionList: FC<GroupedActionListProps> = ({ items }) => {
+  return (
+    <section className="grid grid-cols-1 gap-4 py-2 sm:grid-cols-2">
+      {items.map((item) => (
+        <GroupedActionItem key={item.title} {...item} />
```

Comment:
I think we shouldn't pass arrays of props, and instead just compose the items in rendered (if you'll want to change the color, we can do it when rendering the children elsewhere, or just do the children.map clone shennanigans)